src/nvim/fileio.c

```diff
@@ -6260,75 +6282,70 @@ int trigger_cursorhold(void)
       && !ins_compl_active()
       ) {
     state = get_real_state();
-    if (state == NORMAL_BUSY || (state & INSERT) != 0)
-      return TRUE;
+    if (state == NORMAL_BUSY || (state & INSERT) != 0) {
+      return true;
+    }
   }
-  return FALSE;
+  return false;
 }
 
-/*
- * Return TRUE when there is a CursorMoved autocommand defined.
- */
-int has_cursormoved(void)
+/// Return true when there is a CursorMoved autocommand defined.
```

Comment:
It might be a change for a later PR, but why not
```
boll has_event(int event)
{
  return first_autopat[event] != NULL;
}
```
instead of these 6 functions